dietpi/dietpi-drive_manager

```diff
@@ -124,15 +124,23 @@
 			for i in /dev/sd[a-z][0-9]
 			do
 
-				aDRIVE_MOUNT_SOURCE+=($i)
+				if [[ -b $i ]]; then
```

Comment:
@Fourdee 
Doesn't the existence of `/dev/sdX` already prove the existence of the block device? Or are there cases e.g. `/dev/sdb1` shows up but is no block device?

Reply:
@MichaIng 

Yep, it does, good spot. 👍  I'll send commit:
```
[ INFO ] DietPi-Drive_Manager | Detecting drives, please wait...
array created /dev/sda1
array created /dev/mmcblk1p1
array created /dev/mmcblk1p2
```